File: ABC/ABC264/B.cs
using System;
using System.Linq;

namespace Atcoder
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] nums = Console.ReadLine().Split().Select(int.Parse).ToArray();
            int r = nums[0];
            int c = nums[1];
            if(Math.Max(Math.Abs(8-r), Math.Abs(8-c))%2 == 0)
            {
                Console.WriteLine("white");
            } else {
                Console.WriteLine("black");
            }
        }
    }
}

File: ABC/ABC264/rust/B.rs
use proconio::input;
use std::cmp;

fn main() {
    input! {
        r: i32, c: i32
    }
    if cmp::max((8-r).abs(), (8-c).abs()) % 2 == 0 {
        println!("white");
    } else {
        println!("black");
    }
}